src/binding/python/Series.cpp

```diff
@@ -169,6 +169,7 @@ void init_Series(py::module &m) {
         .def("flush", &Series::flush)
 
         .def_property_readonly("backend", &Series::backend)
+        .def("backend_property", &Series::backendProperty)
```

Comment:
Just wondering if we need a new `series` property. We already have `openPMD::getFileExtensions()`/`openPMD_api.file_extensions` where we still need to add `.sst`, I think.

Do you have concrete additional plans to use this new property for?

Reply:
Didn't think of `getFileExtensions()`, I can use that one too.